notes-service/src/test/java/org/exoplatform/wiki/jpa/dao/DraftPageDAOTest.java

```diff
@@ -346,6 +348,7 @@ public void testFindLatestDraftPageByTargetPage() {
     dp1.setCreatedDate(new Date());
     dp1.setUpdatedDate(new Date());
     draftPageDAO.create(dp1);
+    persist();
```

Comment:
this is necessary ?

Reply:
yes, that fixes a random fail in that test